libraries/SITL/SITL.h

```diff
@@ -155,6 +155,7 @@ class SIM {
         GPS_HEADING_HDT  = 1,
         GPS_HEADING_THS  = 2,
         GPS_HEADING_KSXT = 3,
+        GPS_HEADING_BASE = 4,  // nb.  I'm guessing, this could be ROVER!
```

Comment:
needs documenting

Reply:
Fixed this up - the enumeration name was correct.